data/saas/config/aircall_config.yml

```diff
@@ -0,0 +1,69 @@
+saas_config:
+  fides_key: <instance_fides_key>
+  name: Aircall SaaS Config
+  type: aircall
+  description: A sample schema representing the Aircall connector for Fides
+  version: 0.1.0
+
+  connector_params:
+    - name: domain
+      default_value: api.aircall.io
+    - name: api_token
```

Comment:
This connector uses `basic` authentication so you should ask the user for `api_id` and `api_token`, these are used as the username and password.

```
  connector_params:
    - name: domain
      default_value: api.aircall.io
    - name: api_id
      label: API ID
    - name api_token
      label: API Token

  client_config:
    protocol: https
    host: <domain>
    authentication:
      strategy: basic
      configuration:
          - username: <api_id>
          - password: <api_token>
```